src/plugins/prebuilt-library/ai-wizard/kadence-ai-wizard.js

```diff
@@ -98,9 +101,16 @@ export function KadenceAiWizard( props ) {
 
 		if (saveStatus) {
 			onWizardClose( 'saved' );
+			handleEvent( 'ai_wizard_update' );
```

Comment:
Maybe this would be better as a constant [JS constant](https://github.com/stellarwp/kadence-blocks/blob/feature/KADEN-287/ai-wizard-completed/src/plugins/prebuilt-library/ai-wizard/constants/index.js)?